contrib/openldap/template.py

```diff
@@ -0,0 +1,70 @@
+pkgname = "openldap"
+pkgver = "2.6.8"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--disable-debug",
+    "--disable-slapd",
+    "--disable-sql",
+    "--disable-static",
+    "--disable-versioning",
+    "--disable-wt",
+    "--enable-backends=mod",
+    "--enable-crypt",
+    "--enable-dynamic",
+    "--enable-modules",
+    "--enable-overlays=mod",
+    "--enable-rlookups",
+    "--enable-spasswd",
+    "--enable-syslog",
+    "--with-cyrus-sasl",
+    "--with-tls=openssl",
+]
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "gmake",
+    "libtool",
+    "mandoc",
+    "pkgconf",
+]
+makedepends = [
+    "libevent-devel",
+    "libltdl-devel",
+    "libsasl-devel",
+    "libsodium-devel",
+    "openssl-devel",
+]
+depends = [f"openldap-progs={pkgver}-r{pkgrel}"]
+pkgdesc = "FOSS implementation of the Lightweight Directory Access Protocol"
+maintainer = "Renato Botelho do Couto <renato@netgate.com>"
+license = "OLDAP-2.8"
+url = "https://www.openldap.org"
+source = (
+    f"{url}/software/download/OpenLDAP/{pkgname}-release/{pkgname}-{pkgver}.tgz"
+)
+sha256 = "48969323e94e3be3b03c6a132942dcba7ef8d545f2ad35401709019f696c3c4e"
+# test suite takes ages
+options = ["!check"]
```

Comment:
afaik this is mostly the server part, but maybe there's library-only tests that are more reasonable?